cf_units/__init__.py

```diff
@@ -1883,27 +1887,26 @@ def convert(self, value, other, ctype=FLOAT64):
             32.0
             >>> a64 = np.arange(10, dtype=np.float64)
             >>> c.convert(a64, f)
-            array([ 32. ,  33.8,  35.6,  37.4,  39.2,  41. ,  42.8,  44.6, \
- 46.4,  48.2])
+            array([ 32. ,  33.8,  35.6,  37.4,  39.2,  41. ,  42.8,  44.6,
+                    46.4,  48.2])
             >>> a32 = np.arange(10, dtype=np.float32)
             >>> c.convert(a32, f)
             array([ 32.        ,  33.79999924,  35.59999847,  37.40000153,
                     39.20000076,  41.        ,  42.79999924,  44.59999847,
                     46.40000153,  48.20000076], dtype=float32)
 
-        .. note::
-
-            Conversion is done *in-place* for numpy arrays. Also note that,
-            conversion between unit calendars is not permitted.
-
+        .. note:: Conversion between unit calendars is not permitted.
```

Comment:
I think this should still be split over multiple lines like it was previously:

```
.. note::

   Conversion between unit calendars is not permitted.
```

There should also be a blank line after the note before the end of the comment.